### DISCUSSION TITLE
Makes dragged block semitransparent

### DISCUSSION THREAD

**Discussion by sfederici** [ScratchAddons/ScratchAddons#2551] [Ideas]
Snap (snap.berkeley.edu) solved on of the more annoying Scratch problems, that is dragging a reporter block in the right argument, making the dragged block semitransparent. 

Another difference, being the blocks semitransparent while dragged, so that you can easily see the arguments below, is that a reporter fit into an argument while the mouse pointer -not the reporter end, that you cannot always easily grasp- is right _inside_ the argument.

This avoid a lot of frustration to new Scratchers.

**Comment by TheColaber** [comment]
You may know about the new (#2332) Transparent orphan blocks addon.
![image](https://user-images.githubusercontent.com/72760579/119133165-960f7700-ba09-11eb-84d0-4612df8cf276.png)
Currently, if dragging a block, it loses its transparency. Maybe we could add a setting within this addon, being something like "Drag Transparency" so you can maybe make this value 30%, but all orphan blocks could be kept at 0% if need? I don't know if this is such a good idea, because of the addon's name specifying "orphan". But anyway, I wanted to bring this idea to the table.

**Reply by mxmou** [reply]
Maybe the addon could be renamed to just "block transparency" and have settings for normal blocks, orphan blocks and dragged blocks.

**Reply by TheColaber** [reply]
I've created #2552

**Reply by sfederici** [reply]
> Maybe the addon could be renamed to just "block transparency" and have settings for normal blocks, orphan blocks and dragged blocks.

Excellent idea

**Reply by sfederici** [reply]
> You may know about the new (#2332) Transparent orphan blocks addon.

Yes, this is where I got inspired
